src/misc/msccl/msccl_lifecycle.cc

```diff
@@ -470,18 +474,60 @@ ncclResult_t mscclEnqueueCheck(
     count, dataType, root, peer, op, func, comm, stream,
     &threadLocalStatus.savedSchedulerParams.back()));
 
+  //printf("In msccl enqueue check\n");
+  NCCLCHECK(mscclGetCaptureStatus(stream));
```

Comment:
These seem to be unused?

Reply:
It should be removed if unused.

Reply:
Looks like this was added before my changes. Do you recall if it was necessary? I'll try removing and redo some tests.